src/proto/grpc/testing/control.proto

```diff
@@ -169,3 +169,18 @@ message CoreResponse {
 
 message Void {
 }
+
+message Scenario {
```

Comment:
We'll want some comments for the message before we submit this (we suffer from lack of documentation in performance related protos in general, so let's not make this worse).